### PR TITLE
CSI-3103 add replication deployment

### PR DESCRIPTION
**Pull request by oriyarde**
https://github.com/csi-addons/volume-replication-operator/

### CODE REVIEW

File: pkg/controller/ibmblockcsi/syncer/csi_controller.go [hunk 1]
```diff
@@ -165,6 +166,15 @@ func (s *csiControllerSyncer) ensureContainersSpec() []corev1.Container {
 	)
 	resizer.ImagePullPolicy = s.getCSIResizerPullPolicy()
 
+	leaderElectionNamespaceFlag := fmt.Sprintf("--leader-election-namespace=%s", s.driver.Namespace)
```
Comment (zingero) [inline pkg/controller/ibmblockcsi/syncer/csi_controller.go:169]: The other sidecars have comments before the `ensureContainer` call and a "TODO" comment also.
I suggest adding similar comments or removing the other comments.
Personally, I prefer removing the comments.

**Review by ArbelNathan — APPROVED** [review]
LGTM GJ

File: pkg/controller/ibmblockcsi/syncer/csi_controller.go (hunk 1 above)
Comment (oriyarde) [reply pkg/controller/ibmblockcsi/syncer/csi_controller.go:160]: done

**Review by ArbelNathan — DISMISSED** [review]
LGTM

**Review by zingero — DISMISSED** [review]

**Review by ArbelNathan — APPROVED** [review]
GJ

**Review by matancarmeli7 — APPROVED** [review]
LGTM

**Review by zingero — APPROVED** [review]

**Pull request merged**